src/styles.scss

```diff
@@ -2,60 +2,29 @@
 
 // Better style reset
 @import "~normalize.css/normalize";
-
 // A cleaner way to do breakpoints/media queries, http://breakpoint-sass.com/
 @import "~breakpoint-sass/stylesheets/breakpoint";
```

Comment:
Is this what provides the `breakpoint` include that we have been using in the SCSS files?

Reply:
Correct.